integration/test_magic_folder.py

```diff
@@ -460,3 +464,57 @@ def test_alice_adds_files_while_bob_is_offline(reactor, request, temp_dir, magic
             in conflict_files
         ),
     )
+
+
+@pytest_twisted.inlineCallbacks
+def test_many_files_progress_api(reactor, request, magic_folder, alice, bob):
+    """
+    cypher reports that weird things happen with the status-API,
+    sometimes, when you add a magic-folder with a ton of files.
+    """
+
+    alice_magic_dir, bob_magic_dir = magic_folder
+
+    data = urandom(1024) * 1024 * 1  # 20 megabyte files
```

Comment:
Looks like a one megabyte file.  I guess the point of making it larger is to be able to catch it in the act?  Files too small might be done before the test suite gets around to asking about the status..  Except ... the assertions below don't seem to care.  They just want 50 upload/download/success items before aging-out and 0 after.  So maybe fewer, smaller files would be okay?  Or maybe the test isn't done.

If the point is to make processing the files expensive and slow there's still a chance the test could lose some race since file size doesn't guarantee anything about kernel scheduling of the difference processes involved.